lib/NQP/Config.pm

```diff
@@ -423,13 +423,13 @@ sub configure_jars {
 sub configure_relocatability {
     my $self = shift;
 
-    # Relocatability is not supported on AIX.
-    if ( $^O =~ /^(?:aix|openbsd)$/ ) {
-        $self->{config}->{relocatable} = 'nonreloc';
-    }
-    else {
-        $self->{config}->{relocatable} = $self->{options}->{relocatable} ? 'reloc' : 'nonreloc';
+    # Relocatability is not supported on AIX and OpenBSD.
+    if ( $^O =~ /^(?:aix|openbsd)$/ && $self->{options}->{relocatable} ) {
+        die 'Relocatability is not supported on ' . $^O .
```

Comment:
There is method `sorry` for this. Makes nicer less annoying error messages. ;)

Reply:
Done.